java/src/main/java/com/logicalclocks/hsfs/metadata/StorageConnectorApi.java

```diff
@@ -63,15 +64,17 @@ public StorageConnector getOnlineStorageConnector(FeatureStore featureStore)
     HopsworksClient hopsworksClient = HopsworksClient.getInstance();
     String pathTemplate = HopsworksClient.PROJECT_PATH
         + FeatureStoreApi.FEATURE_STORE_PATH
-        + ONLINE_CONNECTOR_PATH;
+        + CONNECTOR_PATH;
 
     String uri = UriTemplate.fromTemplate(pathTemplate)
         .set("projectId", featureStore.getProjectId())
         .set("fsId", featureStore.getId())
         .expand();
 
     LOGGER.info("Sending metadata request: " + uri);
-    return hopsworksClient.handleRequest(new HttpGet(uri), StorageConnector.class);
+    StorageConnector[] storageConnectors =  hopsworksClient.handleRequest(new HttpGet(uri), StorageConnector[].class);
+    return Arrays.stream(storageConnectors).filter(s -> s.getName().contains(ONLINE_FEATURE_STORE_CONNECTOR_SUFFIX))
```

Comment:
can you please double check if the backend lower cases the connector name before sending it? or if we even allow mixed case storage connector names.

Reply:
The online connector name has a specific format. It is project_name + username + ONLINE_FEATURE_STORE_CONNECTOR_SUFFIX. 
In this PR for backend we created the following query @NamedQuery(name = "FeaturestoreJdbcConnector.findByName", query = "SELECT fsjdbc FROM FeaturestoreJdbcConnector " +
        "fsjdbc WHERE lower(fsjdbc.name)  = lower(:name)")}) so that they are unique regardless of case

Reply:
yes sorry, I misread the code, I thought you are filtering by the entire name, not just the suffix.